agnosterzak.zsh-theme

```diff
@@ -89,6 +89,53 @@ prompt_context() {
 prompt_battery() {
   HEART='♥ '
 
+  if [[ "$OSTYPE" = darwin* ]] ; then
+
```

Comment:
Just a very minor appreciation: `if [[ $(uname) == "Darwin"  ]] ; then` works as well, and looks similar than the basecode line for Linux systems :)